CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+<!--
+SPDX-FileCopyrightText: Copyright 2025 Arm Limited and affiliates.
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# CONTRIBUTING
+
+Thank you for your interest in contributing to ML Tool Solutions.
+This document describes the recommended development workflow, coding standards, and how to use the repository's automated tooling, particularly the pre-commit hooks that enforce SPDX/REUSE compliance, formatting checks, and other quality gates.
+
+We welcome contributions through GitHub Pull Requests.
+Please read this guide before submitting changes.
+
+## Development Workflow
+
+### Fork, Branch, and Submit a PR
+
+1. Fork the repository.
+1. Create a feature branch:
```

Comment:
I think
1.
1.
1.
etc. is ok from a MD rendering point of view - but it makes the MD harder to read as src than...
1.
2.
3.
etc.

Reply:
Done